pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -240,6 +240,10 @@ type AlertmanagerSpec struct {
 	HostAliases []HostAlias `json:"hostAliases,omitempty"`
 	// Defines the web command line flags when starting Alertmanager.
 	Web *AlertmanagerWebSpec `json:"web,omitempty"`
+
+	// Defines the cluster tls configuration for Alertmanager mTLS gossip
```

Comment:
```suggestion
	// Configures the mutual TLS configuration for the Alertmanager cluster's gossip protocol.
```

Reply:
We should also reflect that it's an experimental feature.
Also worth documenting that it requires Alertmanager >= 0.24.0.